docs/CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ Contributing to NeoForge
    - If possible, write an automated test under the tests project. See [NEOGAMETESTS.md](NEOGAMETESTS.md) for more info.
 10. Run `gradlew genPatches` to generate patch-files from the patched sources
 11. Run `gradlew applyAllFormatting` to automatically format sources
-12. Check correct formatting with `gradlew spotlessCheck`
+12. Check correct formatting with `gradlew immaculateCheck`
```

Comment:
Should we perhaps recommend that `checkFormatting` task instead, as seen in the `neoforge.formatting-conventions` script plugin? It'd also check the license headers, and would be consistent with what the GitHub Actions runs for the PR checks.

Reply:
Made that change; I agree, it's more sensible.